Algorithm.Framework/Portfolio/PortfolioConstructionModel.cs

```diff
@@ -42,5 +54,17 @@ public virtual IEnumerable<IPortfolioTarget> CreateTargets(QCAlgorithmFramework
         public virtual void OnSecuritiesChanged(QCAlgorithmFramework algorithm, SecurityChanges changes)
         {
         }
+
+        public static double[,] GetReturns(IEnumerable<Dictionary<DateTime, double>> returnsByDate)
```

Comment:
Not sure if this function belongs here. Specifically, it's not *getting* returns, but its massaging the provided returns into a specific structure according to specific rules. Maybe simply changing the name would make this more descriptive? Also, some XML documentation might help here as well.

EDIT: Looks like this may find a better home on `ReturnsSymbolData` ?